config/jobs/kubernetes-sigs/image-builder/image-builder-ci.yaml

```diff
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes-sigs/image-builder-konfigadm:
+    - name: post-image-builder-konfigadm
+      labels:
+        preset-dind-enabled: "true"
```

Comment:
why dind?

Reply:
also the dind preset will not work without executing some pre-reqs.